packages/apptive_grid_core/lib/src/model/form/after_submit_action.dart

```diff
@@ -0,0 +1,101 @@
+/// The AfterSubmitAction class represents an action to be taken after a form
+/// is submitted.
+///
+/// Currently only the custom button title is used in the app.
```

Comment:
This comment should be with the form package as at this point here it is not determined what is done and not done.
Alternatively if we decide to not check for the other thing it could be thought about removing them for now

Reply:
I moved the comment to the `ApptiveGridFormData`. 
And I would argue, it doesn't hurt to already parse the other options, even if we don't use them.